tests/robustness/Makefile

```diff
@@ -136,6 +133,14 @@ $(GOPATH)/bin/gofail: $(REPOSITORY_ROOT)/tools/mod/go.mod $(REPOSITORY_ROOT)/too
 	  $(MAKE) gofail-enable; \
 	  $(MAKE) build;
 
+/tmp/etcd-v3.6.4-failpoints/bin: $(GOPATH)/bin/gofail
```

Comment:
Can you implement it same way we do for any other version?

See line 139
```
/tmp/etcd-v3.5.2-failpoints/bin:
/tmp/etcd-v3.5.4-failpoints/bin:
/tmp/etcd-v3.5.5-failpoints/bin:
/tmp/etcd-v3.5.%-failpoints/bin: $(GOPATH)/bin/gofail
```

Reply:
Fix attempted. Let's see what happens on CI!